.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,60 @@
+name: Feature request
+description: Suggest an idea for Cocos
+labels: ["Needs Triage","Feature Request"]
+body:
+
+- type: markdown
+  attributes:
+    value: |
+      - Please write a descriptive proposal title above.
+      - Search [open](https://github.com/cocos/cocos-engine/issues) and [closed](https://github.com/cocos/cocos-engine/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been suggested.
+
+- type: textarea
+  attributes:
+    label: Use Case
+    description: Explain your use case, context, and rationale behind this feature request. More importantly, what is the project you are trying to build that led to the need for this feature? This is important to know the **context** in which the feature is being proposed.
+    placeholder: Example - "I am working on a MOBA game that ... With heavy use of collision detection..."
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: Problem Description
```

Comment:
I think it can be merged in use case

Reply:
I want to force people to describe the source of the need so that we can assess the generality of the need and whether there is a better solution. For similar projects, it is normal to have a little extra cost when raising need. The same is true for our internal eco service team.